src/main/java/uwu/narumi/deobfuscator/transformer/impl/skidfuscator/SkidfuscatorStringTransformer.java

```diff
@@ -0,0 +1,69 @@
+package uwu.narumi.deobfuscator.transformer.impl.skidfuscator;
+
+import org.objectweb.asm.tree.ClassNode;
+import org.objectweb.asm.tree.LdcInsnNode;
+import org.objectweb.asm.tree.MethodInsnNode;
+import org.objectweb.asm.tree.MethodNode;
+import uwu.narumi.deobfuscator.Deobfuscator;
+import uwu.narumi.deobfuscator.helper.ASMHelper;
+import uwu.narumi.deobfuscator.transformer.Transformer;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Base64;
+import java.util.HashMap;
+
+public class SkidfuscatorStringTransformer extends Transformer {
+    // Goofiest transformer
+    @Override
+    public void transform(Deobfuscator deobfuscator) throws Exception {
+        deobfuscator.classes().forEach(classNode -> {
+            HashMap<ClassNode, byte[]> classToKeys = new HashMap<>();
+
+            new ArrayList<>(classNode.methods).forEach(mn -> Arrays.stream(mn.instructions.toArray())
```

Comment:
```suggestion
            classNode.methods.forEach(mn -> Arrays.stream(mn.instructions.toArray())
```
You can already use forEach on `classNode.methods`, no need to create an ArrayList

Reply:
It's to stop a concurrent modification exception when removing the decrypt method.